docs/about/technical_steering_committee.md

```diff
@@ -1,55 +1,126 @@
 # Kedro's Technical Steering Committee
 
-Kedro is an incubating project within [LF AI & Data](https://lfaidata.foundation/).
+Kedro is a graduated project within [LF AI & Data](https://lfaidata.foundation/).
```

Comment:
```suggestion
Kedro is a graduate-stage project within [LF AI & Data](https://lfaidata.foundation/).
```
Nit: I think this is the right terminology; see https://lfaidata.foundation/projects/kedro/.